### DISCUSSION TITLE
How to intercept automatic pasting when the "separator" property is set.

### DISCUSSION THREAD

**Discussion by DevExplotel** [primefaces/community#1353] [PrimeNG]
Hello, I need to intercept what will be pasted in order to authorize the pasting or not. I manage to do this only when I don't define separators. As soon as I define a separator, my directive loses control of the input field and so I can paste anything.
Thanks

**Comment by cetincakiroglu** [comment]
Hi,

Could you please share a stackblitz example so we can identify the issue?

**Comment by cetincakiroglu** [comment]
Closing the discussion due to inactivity.
